src/dds/qos.rs

```diff
@@ -75,36 +74,43 @@ impl QosPolicyBuilder {
     Self::default()
   }
 
+  #[must_use]
```

Comment:
I think in the case of a 'builder' type it's better to mark the whole type as `must_use` rather than the individual methods. There's currently a clippy false positive for this case, but that can be safely ignored

Reply:
Yes, but then clippy complains and there is no green check mark.

Let's leave it as as workaround to contradictory clippy until the clippy is fixed.